ups/ctrl_rucio_ingest.table

```diff
@@ -0,0 +1,21 @@
+# List EUPS dependencies of this package here
```

Comment:
Nothing here should be needed; I don't think you even want this to be an eups package.

Reply:
Removed